src/main/resources/db/changelog/20220722000000-add_delete_cascade_to_consumer_fks.xml

```diff
@@ -0,0 +1,104 @@
+            constraintName="fk_cp_consumer_facts_consumer"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-5" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_cnsmr_capability_cnsmr"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_capability" constraintName="fk_cnsmr_capability_cnsmr"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-6" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_capability"
+            baseColumnNames="consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="fk_consumer_capability_consumer"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-7" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_consumer_guests"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_guests" constraintName="fk_consumer_guests"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-8" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_guests"
+            baseColumnNames="consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="fk_consumer_guests_consumer"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-9" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_installed_product"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_installed_products" constraintName="fk_installed_product"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-10" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_installed_products"
+            baseColumnNames="consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="fk_installed_products_consumer"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-11" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_guest_attribute"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_guests_attributes" constraintName="fk_guest_attribute"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-12" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_guests_attributes"
+            baseColumnNames="cp_consumer_guest_id"
+            referencedTableName="cp_consumer_guests"
+            referencedColumnNames="id"
+            constraintName="fk_consumer_guests_attributes_consumer_guests"
```

Comment:
@Ceiu I added the results of an investigation on max size of constraint names to the description of the PR.

Reply:
Looks like 128bytes is the defacto standard across the three at the moment. I don't see any reason we would use non-ASCII characters for our object names, so 64 characters seems like a safe, reasonable limit.